rootex/framework/scene.cpp

```diff
@@ -7,8 +7,8 @@
 #include "components/visual/camera_component.h"
 #include "components/audio/audio_listener_component.h"
 
-static SceneID NextSceneID = ROOT_SCENE_ID + 1;
```

Comment:
Keep this in scene.cpp, not scene.h
This will avoid NextSceneID from being defined in each of the files where scene.h is included.